client/templates/app/signup/signup.html

```diff
@@ -1,24 +1,5 @@
-<template name="publicSignup">  
+<template name="publicSignup">
   <div class="row">
-    <form class="col s6 offset-s3 signup-form">
-      <h2>Register for Cherish</h2>
-      <div class="row">
-        <div class="input-field col s12">
-          <input id="email" type="email" class="validate email-address-input">
-          <label for="email">Email</label>
-        </div>
-      </div>
-      <div class="row">
-        <div class="input-field col s12">
-          <input id="password" type="password" class="validate password-input">
-          <label for="password">Password</label>
-        </div>
-      </div>
-      <div class="row">
-        <div class="col s12">
-          <button class="btn waves-effect waves-light" type="submit" name="action">Sign up</button>
-        </div>
-      </div>
-    </form>
+  {{> atForm}}
```

Comment:
Can we alter this at all? I'm not a fan of the default look and wouldn't want to commit it just for someone else to re-do it.